apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/build-default-renderer.tsx

```diff
@@ -3,4 +3,4 @@ import { IBaseViewModel } from '@coscrad/api-interfaces';
 export const buildDefaultRenderer =
     <T extends IBaseViewModel>(propertyKey: keyof T) =>
     (input: T) =>
-        <div>{(JSON.stringify(input[propertyKey]) || '').replace(/"/g, '')}</div>;
```

Comment:
These extra `<div>`s make things harder to troubleshoot, so we should just use `<>` fragments wherever possible and divs only if actually adding important function.